src/frontend/efiling-frontend/src/components/page/cso-account/CSOAccount.scss

```diff
@@ -2,7 +2,30 @@
   * {
     outline: none !important;
   }
-  .error {
+  .email-form {
+    margin-left: 75px;
+  }
+  .email-input {
+    margin-bottom: 5px;
+  }
+  .cso-label {
+    font-weight: bold;
+    margin: 0 0 3px 0;
+  }
+  .cso-input {
+    border: 1px solid;
+    width: 330px;
+  }
+  .red {
     color: red;
   }
-}
+  .error {
+    margin-left: 5px;
+  }
+
+  @media (max-width: 430px) {
+    .cso-input {
+      width: 100%;
+    }
```

Comment:
I think it might be better to use the pre-defined bootstrap breakpoints rather than overriding them.  430px is not one of the known breakpoints. 
https://getbootstrap.com/docs/4.0/layout/grid/
ie. instead of `cso-input` as the class, I believe you could use `col-sm-12` on the input field (12 is the maximum number of columns, thus 100%).

Reply:
I did not change this. probably the linter, not sure you want me to look at this...